.github/instructions/test.instructions.md

```diff
@@ -0,0 +1,25 @@
+<!-- ~~ Generated by projen. To modify, edit .projenrc.ts and run "npx projen". -->
+
+---
+applyTo: "**/*.test.ts,**/*.spec.ts"
+---
+
+# Testing Instructions
+
+## Testing Guidelines
+
+### Test Structure
```

Comment:
The H2 isn't needed at all, adjust in CopilotInstructions. 

All of the H3s can become H2s